### DISCUSSION TITLE
You don't have access to GitHub Copilot.

### DISCUSSION THREAD

**Discussion by lucids-git-goods** [community/community#13823] [Copilot Conversations] [answered]
I keep getting this message in Output if I try to Sign In to Copilot from Visual Studio Community 2022 (64 bit) Version 17.1.2:
`You don't have access to GitHub Copilot.`

I have multiple github accounts, but I made sure the one signed into Visual Studio is the one that already has access. 

I'm not sure if it's related, but every time I start Visual Studio under File | Account Settings it says:
`We need to refresh the credentials for this account.  `
under this GitHub account.  When I refresh the credentials it says it was successful, and the messages disappears, but I still get the 'You don't have access to GitHub Copilot' message after that.

Edit: I ended up getting the welcome message from my other Github account, and that one works fine.  I couldn't find anywhere I had linked the account to my Microsoft Visual Studio developer profile, but if I had linked either of the two accounts, it would have been the one that ended up working.

**Comment by mentoster** [comment, marked as answer]
I solve this with this [discussion](https://github.com/github/feedback/discussions/13982)
